docs/src/cli/commands.md

```diff
@@ -54,6 +54,12 @@ anchor build
 
 Builds programs in the workspace targeting Solana's BPF runtime and emitting IDLs in the `target/idl` directory.
 
+```
+anchor build --verifiable
+```
+
+Runs the build inside a docker image so that the output binary is deterministic (assuming a Cargo.lock file is used).
```

Comment:
```suggestion
Runs the build inside a docker image so that the output binary is deterministic (assuming a Cargo.lock file is used). This command must be run from within a single crate subdirectory within the workspace. For example, `programs/<my-program>/`.
```